pkg/apis/extensions/types.go

```diff
@@ -30,27 +30,26 @@ package extensions
 
 import (
 	"k8s.io/kubernetes/pkg/api"
-	"k8s.io/kubernetes/pkg/api/resource"
 	"k8s.io/kubernetes/pkg/api/unversioned"
 	"k8s.io/kubernetes/pkg/util"
 )
 
-// ScaleSpec describes the attributes a Scale subresource
+// ScaleSpec describes the attributes of a Scale subresource
 type ScaleSpec struct {
-	// Replicas is the number of desired replicas. More info: http://releases.k8s.io/HEAD/docs/user-guide/replication-controller.md#what-is-a-replication-controller"
+	// Replicas is the number of desired replicas. Setting this number will set the number of the desired replicas in the underlying replication controller/deployment.
 	Replicas int `json:"replicas,omitempty"`
 }
 
 // ScaleStatus represents the current status of a Scale subresource.
 type ScaleStatus struct {
-	// Replicas is the number of actual replicas. More info: http://releases.k8s.io/HEAD/docs/user-guide/replication-controller.md#what-is-a-replication-controller
+	// Replicas is the number of actual replicas in the underlying replication controller/deployment.
```

Comment:
Rather than enumerate replication controller or deployment explicitly, is it better to just state:

"Replicas is the actual number of observed instances of the scaled object"

I worry no one will go back and update this text if/when we change or add additional scale support to other objects.